Text analysis slides - review required

Irene-Berezin opened the pull request.

Irene-Berezin committed:
added outline

Irene-Berezin committed:
added references

Alexr951 committed:
Added section 2.3

Alexr951 committed:
Update text_analysis.qmd

Irene-Berezin committed:
Merge pull request #33 from ubcecon/alexr_text_analysis

Text Analysis - Added section 2.3

Irene-Berezin committed:
added preprocessing text

Irene-Berezin committed:
complete 2.1 + examples

Irene-Berezin committed:
section 2.3

Irene-Berezin committed:
added footnotes and sources

Irene-Berezin committed:
typo fixing

Irene-Berezin committed:
Merge pull request #34 from ubcecon/text_analysis_irene

section 2.1 - review from alex

Irene-Berezin committed:
resolving merge conflicts

Irene-Berezin committed:
Merge pull request #39 from ubcecon/text_analysis_irene

resolving merge conflicts

Irene-Berezin committed:
added images from previous prs

Irene-Berezin committed:
Merge pull request #40 from ubcecon/text_analysis_images

added images from previous prs

Alexr951 reviewed (commented):
@krishaant  Have a look at my comments and let me know if you need anything else.

NathanPalaiologos committed:
made the slides look better

jlgraves-ubc committed:
Merge branch 'main' into text_analysis_overview

jlgraves-ubc commented:
Is this otherwise ready-to-go @Irene-Berezin ?

jlgraves-ubc committed:
Merge branch 'main' into text_analysis_overview

jlgraves-ubc committed:
Update image paths and formatting in docs

Changed image paths from 'images/' to 'media/' and updated image formatting for consistency across documentation files. Added 'status: draft' to deep learning and transformers intros, improved text analysis formatting, and added a 'Coming soon!' note to dissemination page. Fixed relative path for quickstart launch icon.
